client/src/components/productDetails/DefaultGallery.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import MainImage from './MainImage.jsx';
+import ImageThumbnails from './ImageThumbnails.jsx';
+import ArrowButtons from './ArrowButtons.jsx';
+import GalleryList from './GalleryList.jsx';
+
+const { useState, useEffect } = React;
+
+function DefaultGallery({ photoUrls, thumbnailUrls }) {
+  const [count, setCount] = useState(0);
+  const photoAmount = photoUrls.length - 1;
```

Comment:
is this assigning photoAmount to the total number of pictures? if so, would it be photoUrls.length? unless you are assigning it the index, then please disregard this comment.